src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Logo from "../logo/Logo";
+import { MenuItems } from "./MenuItems";
+import styles from "./Navbar.module.scss";
+
+// import * as FaIcons from "react-icons/fa";
+
+function Navbar() {
```

Comment:
As for me, it is better to use whenever it is possible a named export, not default.